static/app/components/events/suspectCommits.tsx

```diff
@@ -33,6 +39,13 @@ export function SuspectCommits({group, eventId, project, commitRow: CommitRow}:
   });
   const committers = data?.committers ?? [];
 
+  const hasStreamlinedUI = useHasStreamlinedUI();
+
+  const [suspectCommitDismissed, setSuspectCommitDismissed] = useLocalStorageState(
+    `suspect-commit-dismissed-${eventId}`,
+    false
+  );
```

Comment:
TBH i don't really get why we'd let people dismiss suspect commits, its pretty different than the rest of the promos. Maybe if they hadn't set it up already? Either way right now what happens is the dismiss doesn't actually hide the commits, just displays the old UI.

Reply:
Also event IDs aren't guarenteed unique for different projects, so the local storage keys could collide, though I think we just shouldn't allow dismissing it in general, just the promo banner (which wouldn't need to be keyed on anything i think)

Reply:
will add project to make the key more unique, showing the old UI on dismiss is just a bug in my conditional. will fix. i think for now, especially since it's not a binding change (it's easy to un-dismiss as we test), ill leave dismissing in. checked with vu and that seems to be the direction we want to move in.